### DISCUSSION TITLE
Length parameter must be greater than 0 in core.php

### DISCUSSION THREAD

**Discussion by BogusCurry** [datenstrom/community#222] [Report a bug] [answered]
Hello ;D
At First ;D Happy New Year ;D

So I have a message on my website .. 
Warning: fread(): Length parameter must be greater than 0 in /kunden/xxxxx/hosting/webseiten/eigene/pinguinsreisende/system/plugins/core.php on line 2827

I not know why, have only two plugins inside .. markbar and streamlist, streamlist have deaktived, the message is ever there. Maybe the PHP Version is change from my hoster. Yellow version is the last actual ;D

Have a nice Day ;D
Bogus

Founder from GridTalk.de

**Comment by markseuffert** [comment, marked as answer]
Thanks Bogus for your report. For some reasons the PHP developers don't like empty files and think that [reading zero bytes is a runtime error](https://github.com/php/php-src/blob/07959c1cae8ef9b69ea6f0586259b7f37939bf4a/ext/standard/file.c#L1809). Have made a workaround in latest Yellow version, should be working now. 

Have a good start into the new year.
